src/presentational/ia-button.ts

```diff
@@ -74,6 +74,7 @@ export class IauxButton extends LitElement {
 
     :host(.secondary) button {
       background: #333;
+      color: #ffeeee;
```

Comment:
Hey @isa!
I know you tried to build a new button as reusable input element but the button styling does not match what we are using for button behaviour or button states like :action, :hover and :focus. 

The latest styling from ia-button is here: `/home/neeraj/iaux-collection-browser/src/styles/ia-button.ts`. This file carry all the button styling we are using most of the new work. It would be good if you can implement that stying here in this component. That way we can feel some behaviour with our buttons.

Here is our button styling [initial ticket](https://webarchive.jira.com/browse/WEBDEV-4929) we worked on and create a ia-button.ts file to use when needed.

Reply:
Also we should host `ia-button`.ts a styling component which be used other places by just importing the package instead of moving this file across repos.

Reply:
hi neeraj yes absolutely! let's fold this in